cafebabel/templates/articles/tags/detail.html

```diff
@@ -1,25 +1,29 @@
 {% extends "base.html" %}
 
+{% block extra_js %}<script src={{ url_for('static', filename='js/parallax.js') }}></script>{% endblock %}
```

Comment:
this parallax is very specific to tags. It may be interesting to rename it accordingly (_tag-parallax.js_?)

Reply:
hum ok !